### ISSUE TITLE
Blotware uninstall 

### ISSUE THREAD

**Issue by Yogeshkumar1st** [0x192/Universal-Android-Debloater#689]
**Your phone:** <phone model>

**Packages:**
```
com.this.is.a.bad.application
com.this.is.another.bad.application
...
```
- [ ] **I removed all those packages on my phone**
If not why. Leave the brackets blank and explain why.

## Document each package the best you can
**List**: `Google`|`Misc`|`Oem` (manufacturer)|`Aosp`|`Pending`|`Carrier` (isp).
**Removal**: `Recommended`, `Advanced`, `Expert` (this can break important features), `Unsafe` (this can bootloop the phone or break extremely important features).

### \<package name\>
**List**: \<list\> 
**Removal**: \<recommendation list\>
> Description. Link to its Playstore page if it exists.

**Comment by AnonymousWP** [comment]
Mate you didn't even fill in the template.

**Comment by Logan1127** [comment]
Other options?

**Comment by Rafee-M** [comment]
To be closed since there's no update from author?

**Comment by AnonymousWP** [comment]
Yes, but nobody but @0x192 can close issues. This project is bleeding to death like how it's going right now. Feels like I'm filing PRs for nothing. Barely anyone can run a large project on its own.

**Comment by Rafee-M** [comment]
Oh my bad. I thought you were a maintainer as well. I really appreciate all that you're doing for the project. I really hope the project gets back on it's feet soon

**Comment by AnonymousWP** [comment]
> Oh my bad. I thought you were a maintainer as well. I really appreciate all that you're doing for the project. I really hope the project gets back on it's feet soon

Nope, but I'd like to. Pretty sure he said he doesn't want to add maintainers (with minimal permissions) though, so yeah. And thanks, trying my best when I have time. I'll look into the pending replies later.
